frontend/src/serviceWorker.ts

```diff
@@ -64,32 +64,7 @@ function registerValidSW(swUrl: any, config: any) {
           return;
         }
         installingWorker.onstatechange = () => {
-          if (installingWorker.state === "installed") {
```

Comment:
not sure of the best way to test this file 🤔

Reply:
Hahah I ended up doing that since I went down a bit of a rabbit hole with removing the serviceWorker. Do you think it would be worthwhile to add as a ticket to investigate removing that since we aren't using it in the app? 🤔

Reply:
I don't think so. I feel that ticket will not get aligned and will be forgotten in the backlog as that file is easy to spot as not being used. With dead code is more important the clean up because the only way to know that is indeed death code is to spent time investigating, so just to save dev's valuable time is worth cleaning up IMO.

So I would not create a clean up ticket for this file for now.